addons/base_setup/res_config.py

```diff
@@ -28,6 +28,10 @@ class base_config_settings(osv.osv_memory):
             help="Share your partners to all companies defined in your instance.\n"
                  " * Checked : Partners are visible for every companies, even if a company is defined on the partner.\n"
                  " * Unchecked : Each company can see only its partner (partners where company is defined). Partners not related to a company are visible for all companies."),
+        'group_multi_currency': fields.boolean('Allow multi currencies',
+            implied_group='base.group_multi_currency',
+            help="Allows you multi currency environment"),
```

Comment:
I bet the "you" is not required. And I don't mean that you are not required. You know I love you.